p4-16/spec/P4-16-spec.mdk

```diff
@@ -3900,7 +3900,7 @@ lp.learn( { hdr.ethernet.srcAddr, hdr.ipv4.src } );
 
 A list may be used to initialize a structure if the list has the same
 number of elements as fields in the structure. The effect of such an
-initializer is to assign to the i-th element of the list to the i-th
+initializer is to assign the nᵗʰ element of the list to the nᵗʰ
```

Comment:
AFAIK, this code is not HTML.  It`s some weird thing call "Madoko" [<http://madoko.org/reference.html>].

However, your implied point is well-taken that checking the spec. of the language in question isn`t a bad idea.  Here is what Madoko has to say about superscripts & subscripts:  http://madoko.org/reference.html#sec-sub--and-super-script

Checking my self-compiled PDF, I found that Madoko did not "like" what I did here with "nᵗʰ".  Using Unicode® for this may have worked if I had written it as e.g. `n&#`_something_`;&#`_some_other_thing_`;` [<http://madoko.org/reference.html#sec-unicode>], but that`s even _less_ readable than doing it the Madoko way IMO.

Thanks for the bug catch.  I will change this around soon-ish and re-upload [or maybe just pushing to my relevant branch will update this PR].

Reply:
Fixed in my WIP.

![image](https://user-images.githubusercontent.com/3038439/167752658-01c0a9ad-fcd9-4e30-8c21-b5f625885318.png)